### ISSUE TITLE
HPACK support

### ISSUE THREAD

**Issue by TojNALvViV** [vinniefalco/Beast#387]
I guess this blocked on HTTP/2 support, but it would be nice to have HPACK support as part of it.

**Comment by vinniefalco** [comment]
This is in the works, see https://github.com/vinniefalco/Beast/issues/122

I'll close this since its a duplicate. Thanks for your support!
